internal/service/assessment_test.go

```diff
@@ -49,11 +49,11 @@ var _ = Describe("assessment service", Ordered, func() {
 			assessment1ID := uuid.New()
 			assessment2ID := uuid.New()
 			assessment3ID := uuid.New()
-			tx := gormdb.Exec(fmt.Sprintf(insertAssessmentStm, assessment1ID, "Test Assessment 1", "org1", service.SourceTypeInventory, "NULL"))
+			tx := gormdb.Exec(fmt.Sprintf(insertAssessmentStm, assessment1ID, "Test Assessment 1", "org1", "testuser", service.SourceTypeInventory, "NULL"))
```

Comment:
This is just added here to the DB but its not really validated
Its relevant for all tests in this file